ui/app/AppLayouts/Wallet/controls/FollowingAddressesDelegate.qml

```diff
@@ -0,0 +1,162 @@
+import QtQuick
+import QtQuick.Controls
+
+import utils
+
+import StatusQ
+import StatusQ.Controls
+import StatusQ.Components
+import StatusQ.Core
+import StatusQ.Core.Theme
+import StatusQ.Core.Utils as StatusQUtils
+import StatusQ.Popups
+
+import shared.controls
+import shared.popups
+import shared.stores as SharedStores
+
+import "../popups"
+import "../controls"
+import "../stores"
+import ".."
+
+import AppLayouts.Wallet.stores as WalletStores
+
+StatusListItem {
+    id: root
+
+    property SharedStores.NetworkConnectionStore networkConnectionStore
+    property var activeNetworks
+    property string name
+    property string address
+    property string ensName
+    property var tags
+    property string avatar
+
+    property int usage: FollowingAddressesDelegate.Usage.Delegate
+    property bool showButtons: sensor.containsMouse
+
+    property alias sendButton: sendButton
+    property alias starButton: starButton
+
+    signal openSendModal(string recipient)
+    signal menuRequested(var menuModel)
```

Comment:
please don't "pack" parametes into signle model. It's shorter but harder to read and maintain. Please use explicit list of parameters with their own types and names.